sentry/values.yaml

```diff
@@ -107,54 +98,33 @@ hooks:
         memory: 1Gi
 
 system:
-  url: ''
-  adminEmail: ''
+  url: ""
+  adminEmail: ""
   secretKey: 'icLq77rCyY_qrMMpXa6TQNjkDV6mU!c'
   public: false #  This should only be used if you’re installing Sentry behind your company’s firewall.
 
 mail:
   backend: dummy # smtp
   useTls: false
-  username: ''
-  password: ''
+  username: ""
+  password: ""
   port: 25
-  host: ''
-  from: ''
+  host: ""
+  from: ""
 
 symbolicator:
   enabled: false
 
 auth:
   register: true
 
-postgres:
-  enabled: true
-  database: sentry
-  user: postgres
-  password: i@ZBAsFoU4gDz_qDJVX2W8
-  host: postgres
-  port: 5432
-
-rabbitmq:
-  # either use RabbitMQ or Redis as broker
-  enabled: true
-  username: guest
-  password: guest
-  host: rabbitmq-ha
-
-redis:
-  host: redis-master
-  password: ""
-  port: 6379
-  db: 0
-
 kafka:
   host: kafka-confluent
   port: 9092
 
 service:
   name: sentry
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
Curious, why change from `LoadBalancer`?

Reply:
Not everybody uses LoadBalancer. It's an expensive option. I think it's more common to see some sort of an ingress controller being used with service set to ClusterIP. IMO anyway.

Reply:
Makes sense thanks for clarifying